src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorHelpers.cs

```diff
@@ -97,6 +97,21 @@ internal static bool AreLengthsTheSame<T>(scoped in ReadOnlyTensorSpan<T> tensor
         internal static bool AreLengthsTheSame(ReadOnlySpan<nint> lengths1, ReadOnlySpan<nint> lengths2)
             => lengths1.SequenceEqual(lengths2);
 
+        internal static bool IsContiguousAndDense<T>(scoped in ReadOnlyTensorSpan<T> tensor)
+        {
+            // Right most dimension must be 1 for a dense tensor.
+            if (tensor._shape.Strides[^1] != 1)
```

Comment:
Why do we need internals for this?  Couldn't we access the `Strides` member on `ReadOnlyTensorSpan`?

Reply:
Resolving per https://github.com/dotnet/runtime/pull/107097#discussion_r1736798375 - please remove all the use of internal members in these tests.